mountpoint-s3/tests/fuse_tests/mod.rs

```diff
@@ -166,6 +170,17 @@ mod mock_session {
                 .get_object_storage_class(&full_key)
                 .map_err(|e| Box::new(e) as Box<dyn std::error::Error>)
         }
+
+        fn restore_object(&mut self, key: &str, expedited: bool) -> Result<(), Box<dyn std::error::Error>> {
+            let full_key = format!("{}{}", self.prefix, key);
+            self.client
+                .restore_object(&full_key)
+                .map_err(|e| Box::new(e) as Box<dyn std::error::Error>)
+        }
+
+        fn is_object_restored(&mut self, key: &str) -> Result<bool, Box<dyn std::error::Error>> {
+            Ok(true)
```

Comment:
Can we get value from the `MockClient`? should be just a simple function like `get_object_storage_class`.

Reply:
Yep, thanks for spotting this loose end